### ISSUE TITLE
Accessibility - Color Contrast (on Sign In page)

### ISSUE THREAD

**Issue by AVbiser** [zetkin/app.zetkin.org#2590]
**Description:**
Color contrast Issue (Success Criterion 1.4.3 Contrast (Minimum) of WCAG 2.1 accessibility standard)

**Problem:**
White text ("Get involved with your organization") and the smaller paragraph on a red background have insufficient contrast, especially affecting users with low vision.

**Possible solution:**
-Verify the color contrast between the foreground text (white) and background red to ensure it meets at least a 4.5:1 ratio.
-Adjust the red background or use darker shades of red if necessary.

A user should be able to read the text without visibility issues.

**Design specifications**

<img width="1505" alt="Image" src="https://github.com/user-attachments/assets/1ac899cf-6acf-44e6-882f-8ae26ca02363" />

**Comment by AVbiser** [comment]
Suggested solution - visual:

<img width="428" alt="Image" src="https://github.com/user-attachments/assets/5bafadf0-d627-44c9-9f8d-6a126c1d35a7" />
